db/seeds.rb

```diff
@@ -396,3 +396,32 @@ def seed_quantity(item_name, organization, storage_location, quantity)
 # ----------------------------------------------------------------------------
 
 Flipper::Adapters::ActiveRecord::Feature.find_or_create_by(key: "new_logo")
+
+# ----------------------------------------------------------------------------
+# Feedback Messages
+# ----------------------------------------------------------------------------
+# Add some FeedbackMessages to fill up the feedback dashboard
+
+users = User.all
+comments = [
+  "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam faucibus, neque eu mattis varius, eros erat elementum sem, quis volutpat erat tortor quis mauris. In condimentum vel sapien in elementum. Pellentesque porttitor mattis orci eu congue. Donec nec ipsum bibendum dolor tempus pellentesque vel ac ante.",
+  "Integer a molestie tortor. Duis pretium urna eget congue porta. Fusce aliquet dolor quis viverra volutpat.",
+  "Nullam dictum ac lectus at scelerisque. Phasellus volutpat, sem at eleifend tristique, massa mi cursus dui, eget pharetra ligula arcu sit amet nunc."
```

Comment:
@rodolfoasantos hey there -- I'am just catching up on old PRs. I'am happy that you added this seed. I would have used https://github.com/faker-ruby/faker to generate fake data, but this is equally good!